orebfuscator-plugin/src/main/java/net/imprex/orebfuscator/config/OrebfuscatorProximityConfig.java

```diff
@@ -66,8 +67,10 @@ public class OrebfuscatorProximityConfig extends AbstractWorldConfig implements
 		this.frustumCullingProjectionMatrix = new Matrix4f() // create projection matrix with aspect 16:9
 				.perspective(frustumCullingFov, 16f / 9f, 0.01f, 2 * distance);
 
-		this.useRayCastCheck = section.getBoolean("useRayCastCheck",
-				section.getBoolean("useFastGazeCheck", false));
+		this.rayCastCheckEnabled = section.getBoolean("rayCastCheck.enabled",
+				section.getBoolean("useRayCastCheck",
+				section.getBoolean("useFastGazeCheck", false)));
+		this.rayCastCheckOnlyCheckCenter = section.getBoolean("rayCastCheck.onlyCheckCenter", false);
```

Comment:
I would be open to enable this flag by default since it shouldn't decrease _security_ and we only lose some accuracy which is fine I think. I haven't had time to this yet so I'm not sure how big the impact is on accuracy. Any thoughts?

Reply:
I was wrong about one thing. Enabling this option should improve performance. The default is false because before this PR all users used the default check full block. I'm not sure how much of a performance boost. I added this option because some PVP games (like UHC) don't require particularly precise checks.

Reply:
I will leave it off by default for now, maybe will change this in the future though.